python/lsst/ip/isr/isrTask.py

```diff
@@ -2112,7 +2127,7 @@ def maskAndInterpolateDefects(self, exposure, defectBaseList):
         defectBaseList : `List` of `Defects`
 
         """
-        self.maskDefects(exposure, defectBaseList)
+        self.maskDefect(exposure, defectBaseList)
```

Comment:
...how did this work before? *Did* this work before?

Reply:
I don't have a clear answer.